src/Data/ProgrammesDb/Entity/RefIsiteOptions.php

```diff
@@ -63,11 +64,11 @@ class RefIsiteOptions
     private $fileId;
 
     /**
-     * local|admin
+     * local|admin|contact
      *
      * @var string
      *
-     * @ORM\Column(type="string", length=5, nullable=false)
+     * @ORM\Column(type="string", length=7, nullable=false)
```

Comment:
leave this as 35, 7 is only fixing the problem for now. There's no problem having this as a longer string

Reply:
yeah, I thought that as well. Cool, I will.